ansible/roles/datadog/templates/datadog.conf.j2

```diff
@@ -8,7 +8,7 @@ dd_url: https://app.datadoghq.com
 api_key: {{ DATADOG_API_KEY }}
 tags: {% for group, group_hosts in groups.items() -%}
     {%- if inventory_hostname in group_hosts %}group:{{ group }}, {% endif -%}
-{%- endfor %} environment:{{ deploy_env }}
+{%- endfor %} environment:{{ env_name|default(deploy_env) }}
```

Comment:
Aren't we creating separate datadog environment for 100k?

Reply:
eventually we want the environment to be the same so that the datatog metrics etc keep working. The new name is just temporary until we migrate

Reply:
Won't some of metrics get messed up?

Reply:
I don't think so